ql/src/test/queries/clientpositive/vectorized_murmurhash_repeating.q

```diff
@@ -0,0 +1,29 @@
+set hive.llap.io.enabled=false;
+set hive.vectorized.execution.enabled=true;
+
+CREATE TABLE source(id string, id2 string) stored as orc;
+LOAD DATA LOCAL INPATH '../../data/files/2048.orc' overwrite into table source;
+
+-- 2048.orc has 2 columns (id, id2) and 2048 rows.
+-- For the first 1024 rows, both id and id2 are not repeating.
+-- For the last 1024 rows, id is repeating while id2 is not.
+
+-- Test MurmurHashStringColStringCol
+explain
```

Comment:
The simple explain is not very useful in this case since it doesn't tell us which and if the expected murmur implementation is used. `EXPLAIN VECTORIZATION DETAIL` should show all the necessary info.